src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonIncentiveLevelService.kt

```diff
@@ -257,15 +298,18 @@ class PrisonIncentiveLevelService(
     map { it.toDTO() }.toList()
 }
 
-private data class SpendLimits(
+private data class Policy(
   val remandTransferLimitInPence: Int,
   val remandSpendLimitInPence: Int,
   val convictedTransferLimitInPence: Int,
   val convictedSpendLimitInPence: Int,
+
+  val visitOrders: Int,
+  val privilegedVisitOrders: Int,
 )
 
-private val spendLimitPolicy = mapOf(
-  "BAS" to SpendLimits(27_50, 275_00, 5_50, 55_00),
-  "STD" to SpendLimits(60_50, 605_00, 19_80, 198_00),
-  "ENH" to SpendLimits(66_00, 660_00, 33_00, 330_00),
+private val prisonIncentiveLevelPolicies = mapOf(
+  "BAS" to Policy(27_50, 275_00, 5_50, 55_00, 2, 1),
+  "STD" to Policy(60_50, 605_00, 19_80, 198_00, 2, 1),
+  "ENH" to Policy(66_00, 660_00, 33_00, 330_00, 2, 1),
```

Comment:
Note that the policy document is not explicit about how many visit orders each level should allow. This PR does not change the "defaults" that were already set.